src/hera.cpp

```diff
@@ -53,6 +53,9 @@ struct hera_instance : evmc_instance {
   bool use_evm2wasm_js = false;
   bool use_evm2wasm_js_trace = false;
 #endif
+#if HERA_METERING_CONTRACT
```

Comment:
IIRC my idea in the issue was to remove the `ifdef`s entirely and just rely on this runtime option.